mlte/validation/condition.py

```diff
@@ -110,6 +102,7 @@ def __str__(self) -> str:
 
     def __eq__(self, other: object) -> bool:
         """Compare Condition instances for equality."""
+        # TODO: is just names enough? Should we compare args and callback?
```

Comment:
Good question.

Reply:
I guess the answer is, we probably should? But maybe it makes sense to expand this once we tackle the adding a registry of the Value that generated the Condition as well, since that would also expand the comparison anyway. I'll leave a note in that new issue.